fakepkg

```diff
@@ -165,6 +166,13 @@ while true ; do
 		shift
 		;;
 
+		-r|--root)
+		ROOT_DIR=$2
+		[[ ! -d "${ROOT_DIR}" ]] && echo -e "The specified alternate root directory does not exist!" && exit -3
+		ROOT_DIR=$(cd "${ROOT_DIR}"; pwd)
```

Comment:
`readlink -e "${ROOT_DIR}"` should work as well and would get rid of a subshell invocation